WcaOnRails/app/views/competitions/my_competitions.html.erb

```diff
@@ -49,7 +49,22 @@
   </h3>
 
   <%= t '.bookmarked_explanation' %>
-
+  <%= form_tag('/competitions/mine', method: :get, class: "competition-select form-inline #{@display}", id: "bookmarked-competitions-query-form") do %>
```

Comment:
Instead of using a string route, please use the rails URL helpers: `form_tag(my_comps_path, ...etc...)`